server/tests/social.auth.spec.js

```diff
@@ -0,0 +1,77 @@
+import chai, { assert } from "chai";
+import chaiHttp from "chai-http";
+import app from "../index";
+import database from "../models/index";
```

Comment:
These are not assertions, you are testing behaviour, So please use expect. You can keep this in a folder oauth